launcher/src/trusted-extensions.ts

```diff
@@ -0,0 +1,68 @@
+/**********************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+import { env } from 'process';
+import { ProductJSON } from './product-json';
+
+export class TrustedExtensions {
+  async configure(): Promise<void> {
+    console.log('# Configuring Trusted Extensions...');
+
+    if (!env.VSCODE_TRUSTED_EXTENSIONS) {
+      console.log('  > env.VSCODE_TRUSTED_EXTENSIONS is not defined, skip this step');
+      return;
+    }
+
+    try {
+      const extensions: string[] = [];
+
+      const tmp = env.VSCODE_TRUSTED_EXTENSIONS.split(',');
+      for (const e of tmp) {
+        if (e) {
+          extensions.push(e);
+        }
+      }
+
+      if (!extensions.length) {
+        console.log('  > env.VSCODE_TRUSTED_EXTENSIONS does not specify any extension');
+        return;
+      }
+
+      for (const e of extensions) {
+        console.log(`  > add ${e}`);
```

Comment:
does it makes sense to move the log above when an extension name is added to the array?

Reply:
I mean at the moment there is:
- one loop to add an extension
and
- another loop to print a log

Reply:
done